sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -42,23 +42,28 @@
     private Protos.ExecutorInfo.Builder executorBuilder;
     private final PodInstance podInstance;
     private final Map<String, Map<String, String>> lastTaskEnvs;
+    private final boolean useDefaultExecutor;
 
     public PodInfoBuilder(
             PodInstanceRequirement podInstanceRequirement,
             String serviceName,
             UUID targetConfigId,
             SchedulerFlags schedulerFlags,
-            Collection<Protos.TaskInfo> currentPodTasks)
-                    throws InvalidRequirementException {
+            Collection<Protos.TaskInfo> currentPodTasks,
+            Protos.FrameworkID frameworkID,
+            boolean useDefaultExecutor) throws InvalidRequirementException {
         PodInstance podInstance = podInstanceRequirement.getPodInstance();
+        this.useDefaultExecutor = useDefaultExecutor;
```

Comment:
I think it originated with the functions taking fewer params but now that there's 2-3 different members involved it'd probably make sense to move to non-static like you say.